runtime/devnet/Cargo.toml

```diff
@@ -226,16 +225,3 @@ try-runtime = [
 ]
 
 experimental = []
-
-# Enable the metadata hash generation.
-#
-# This is hidden behind a feature because it increases the compile time.
-# The wasm binary needs to be compiled twice, once to fetch the metadata,
-# generate the metadata hash and then a second time with the
-# `RUNTIME_METADATA_HASH` environment variable set for the `CheckMetadataHash`
-# extension.
-metadata-hash = ["substrate-wasm-builder/metadata-hash"]
```

Comment:
Why was the metadata-hash stuff removed?

Reply:
I see it was added to PR #194

Reply:
Removed to follow the base-parachain upgrade changes.